### PR TITLE
Add alelo as easy

### PR DESCRIPTION
**Pull request by lennonrar**

### CODE REVIEW

**Review by erfanghorbanee — APPROVED** [review]

File: _data/sites.json [hunk 1]
```diff
@@ -888,6 +888,17 @@
...
     },
 
+    {
+        "name": "Alelo",
+        "url": "https://www.alelo.com.br/meucartao/duvidas-frequentes",
+        "difficulty": "easy",
+        "notes": "On app, go to 'Settings' -> 'Delete Account'. Follow the requested.",
+        "notes_pt_br": "NO app, vá até 'Opções' -> 'Excluir conta'. Siga o solicitado.",
+        "domains": [
+            "alelo.com"
```
Comment (Kefaku) [inline _data/sites.json:898]: Shouldn't this be `alelo.com.br`?

`alelo.com` and `alelo.com.br` seem to be run by different companies

File: _data/sites.json [hunk 2]
```diff
@@ -888,6 +888,17 @@
         ]
     },
 
+    {
+        "name": "Alelo",
+        "url": "https://www.alelo.com.br/meucartao/duvidas-frequentes",
+        "difficulty": "easy",
+        "notes": "On app, go to 'Settings' -> 'Delete Account'. Follow the requested.",
+        "notes_pt_br": "NO app, vá até 'Opções' -> 'Excluir conta'. Siga o solicitado.",
```
Comment (tupaschoal) [inline _data/sites.json:896]:
```suggestion
        "notes": "On app, go to 'Settings' ⮕ 'Delete Account'. Follow the requested.",
        "notes_pt_br": "NO app, vá até 'Opções' ⮕ 'Excluir conta'. Siga o solicitado.",
```

File: _data/sites.json (hunk 1 above)
Comment (tupaschoal) [inline _data/sites.json:898]:
```suggestion
            "alelo.com.br"
```

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**